tests/ha/iscsi_client.pm

```diff
@@ -0,0 +1,39 @@
+use base "hacluster";
+use testapi;
+
+sub run() {
+   script_run("yast2 iscsi-client");
+   assert_screen "yast2-iscsi-client", 30;
+   send_key 'alt-b'; #start iscsi daemon on Boot
+   send_key 'alt-i'; #Initiator name
+   for ( 1 .. 40 ) { send_key "backspace"; }
+   type_string "iqn.1996-04.de.suse:01:" . get_var("HOSTNAME");
+   assert_screen "yast2-iscsi-client-initiator-name", 30;
+   send_key 'alt-v'; #discoVered targets
+   assert_screen "yast2-iscsi-client-discovered-targets", 30;
+   send_key 'alt-d'; #Discovery
+   assert_screen "yast2-iscsi-client-discovery";
+   send_key 'alt-i'; #Ip address
+   type_string "srv1";
```

Comment:
is this meant to be a variable? I suspect this test is a starting point getting ready to be expanded for multi-machine testing?